tests/ks_initial/testing_parameters.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import pandas as pd
+import itertools
+
+# set parameters
+TEST_DATA_DICT = dict(
+    E_MeV_u=10*np.arange(1, 2, 2),
+    voltage_V=50*np.arange(1, 2),
+    electrode_gap_cm=[0.2],
+    # currently jaffe theory crashes for oxygen and helium
+    particle=["proton"],
+    grid_size_um=np.arange(1, 2),
```

Comment:
I'd suggest avoiding over-using aragne, as it much less readable than a plain list.
```suggestion
    grid_size_um=[1,10],
```